cpp/pypowsybl-cpp/bindings.cpp

```diff
@@ -613,7 +613,8 @@ PYBIND11_MODULE(_pypowsybl, m) {
 
     py::enum_<pypowsybl::NadLayoutType>(m, "NadLayoutType")
             .value("FORCE_LAYOUT", pypowsybl::NadLayoutType::FORCE_LAYOUT)
-            .value("GEOGRAPHICAL", pypowsybl::NadLayoutType::GEOGRAPHICAL);
+            .value("GEOGRAPHICAL", pypowsybl::NadLayoutType::GEOGRAPHICAL)
+            .value("FIXED", pypowsybl::NadLayoutType::FIXED);
```

Comment:
Not really fond of having a "fixed" layout, knowing that the voltage levels with no position would be layout based on the force layout.